lib/rouge/lexers/docker.rb

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class Docker < RegexLexer
+      title "Docker"
+      desc "Dockerfile syntax"
+      tag 'docker'
+      aliases 'docker', 'dockerfile'
+      filenames 'Dockerfile', '*.docker'
+      mimetypes 'text/x-dockerfile-config'
+
+      def self.analyze_text(text)
+        return 0.8 if text =~ /\AFROM\s+.+/im
```

Comment:
With the `/i` here, won't this match python files starting with `from x import y`?